src/client/contextmenu.js

```diff
@@ -68,11 +68,14 @@ export default class ContextMenu {
       }, "CMD+K"],
       ["Browser",     (evt) => {
         this.openComponentInWindow("lively-container", evt).then(comp => {
-          var container = _.last(document.querySelectorAll("lively-container"));
+          let containers = Array.from(document.querySelectorAll('lively-container'))
```

Comment:
Using the new loader w/ Babel 6 one could even use the new function bind syntax making it like functional composition / pipe operator:

```javascript
files::filter(e => e.name.match(...))::sortBy(e => e.name)::last()
```

But I did not get it to work with the legacy boot even when replacing and upgrading babel to babel-standalone etc. So for now it has to be functional:

```javascript
last(sortBy(filter(files, e => e.name.match(...)), e => e.name))
```

Will drop same patches today or next week.